libcxx/include/__availability

```diff
@@ -167,10 +165,8 @@
 // clang-format on
 #    define _LIBCPP_INTRODUCED_IN_LLVM_10 0
 #    define _LIBCPP_INTRODUCED_IN_LLVM_10_ATTRIBUTE                                                                    \
-      __attribute__((availability(macos, strict, introduced = 11.0)))                                                  \
-      __attribute__((availability(ios, strict, introduced = 14.0)))                                                    \
-      __attribute__((availability(tvos, strict, introduced = 14.0)))                                                   \
-      __attribute__((availability(watchos, strict, introduced = 7.0)))
+      __attribute__((availability(macos, introduced = 11.0))) __attribute__((availability(ios, introduced = 14.0)))    \
+      __attribute__((availability(tvos, introduced = 14.0))) __attribute__((availability(watchos, introduced = 7.0)))
```

Comment:
Would you mind keeping the attributes on separate lines? In our distribution, we provide an up-to-date libcxx that will supplant the system one (allowing us to support older deployment targets for much longer), but we need to patch the availability attributes here to [point](https://github.com/conda-forge/libcxx-feedstock/blob/main/recipe/patches/0003-custom-error-message-for-old-sdk.patch) our recipe authors to the instructions how to fix this (and try to [validate](https://github.com/conda-forge/libcxx-feedstock/blob/98bf29b4a106b2f5ba0fed5478be95a8b17263ca/recipe/meta.yaml#L80-L84) this, even if somewhat crudely).

Of course we can just introduce the linebreaks ourselves when patching, but if it's not a big hassle, it would help to keep these attributes on individual lines.